vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.h

```diff
@@ -16,6 +16,19 @@ struct VirtualTextShadowNode final : public ShadowNodeBase {
   TransformableText transformableText{};
 
   void AddView(ShadowNode &child, int64_t index) override;
+
+  bool m_isHighlighted = false;
+  folly::dynamic m_ColorValue = nullptr;
+  size_t m_length = 0;
+
+  typedef struct HighlightData {
+    std::vector<std::unique_ptr<HighlightData>> data;
+    size_t spanIdx = 0;
```

Comment:
can the highlightData objects live in the vector instead of off somewhere on the heap?
IOW, std::vector<HighlightData>

Reply:
I don't think I understand this one - can you explain?

Reply:
instead of `std::vector<std::unique_ptr<HighlightData>>` can this be a `std::vector<HighlightData>`? i.e. make the vector hold the elements instead of just contain pointers to the elements. why do we need these to be unique_ptr's ?